examples/FirebaseRoom_ESP8266/README.md

```diff
@@ -0,0 +1,54 @@
+# Firebase Room
+
+## Software setup
+
+1. Install [Arduino 1.6.9](https://www.arduino.cc/en/Main/Software)
+1. Install [Arduino ESP8266 core](https://github.com/esp8266/Arduino#installing-with-boards-manager)
+1. Download [FirebaseArduino library](https://github.com/googlesamples/firebase-arduino/archive/master.zip)
+1. Start Arduino
+1. Click `Sketch > Include Library > Add .ZIP Library...`
+1. Choose `firebase-arduino-master.zip` downloaded in step `3.`
+
+## Hardware setup
+
+1. Get a [Wio Link](http://www.seeedstudio.com/wiki/Wio_Link)
+1. Connect:
+- a [Grove - Vibration Motor](http://www.seeedstudio.com/wiki/Grove_-_Vibration_Motor) on `pin 5`
```

Comment:
if you do a " 1." (with a leading space) it will make this other list a sublist of "connect:"